config/coreos/waagent.conf

```diff
@@ -92,3 +92,7 @@ OS.OpensslPath=None
 
 # Enable RDMA management and set up, should only be used in HPC images
 # OS.EnableRDMA=y
+
+# Enable or disable self-update, default is enabled
+# AutoUpdate.GAFamily=Prod
```

Comment:
hmm we should probably comment out the config keys if we specify the default value. also, can you please move AutoUpdate.Enabled above the AutoUpdate.GAFamily?